### PR TITLE
Docs - Add String Encryption Document

### LINKED ISSUE #8427: Document: String-Encryption

### PR DESCRIPTION
**Pull request by enisn**
- Closes #8427

### CODE REVIEW

File: docs/en/String-Encryption.md [hunk 1]
```diff
@@ -1,4 +1,135 @@
...
+> This package is already installed by default with the startup template. So, most of the time, you don't need to install it manually.
+
+If installation is needed, it is suggested to use the [ABP CLI](https://docs.abp.io/en/abp/latest/CLI) to install this package.
+
+### Using the ABP CLI
+
+Open a command line window in the folder of the project (.csproj file) and type the following command:
+
+```bash
+abp add-package Volo.Abp.GlobalFeatures
```
Comment (rqx110) [inline docs/en/String-Encryption.md]:
```suggestion
abp add-package Volo.Abp.Security
```

File: docs/en/String-Encryption.md [hunk 2]
```diff
@@ -1,4 +1,135 @@
...
+
+```bash
+abp add-package Volo.Abp.Security
+```
+
+### Manual Installation
+
+If you want to manually install;
+
+1. Add the [Volo.Abp.Security]([NuGet Gallery | Volo.Abp.Security 4.2.2](https://www.nuget.org/packages/Volo.Abp.Security)) NuGet package to your project:
```
Comment (enisn) [inline docs/en/String-Encryption.md]:
```suggestion
1. Add the [Volo.Abp.Security](https://www.nuget.org/packages/Volo.Abp.Security) NuGet package to your project:
```

**Comment by hikalkan** [comment]
* Please add this on the navigation menu, under the Infrastructure > Object to Object Mapping.
* Please send PR to rel-4.3 branch since this is a document for a shipped feature, not for a feature still on development.

**Comment by enisn** [comment]
I've made an unsuccessful rebase operation, so this branch is no longer developed.

Follow #8634

**Pull request closed without merging**